tests/data/hashes.csv

```diff
@@ -1,15 +1,8 @@
-default_author/contracts/stub_0,QmeLYjfy6VxvkErTxTc5Gm1MRHH4rkP1E715aQ693PnS6y
```

Comment:
Why the diff?

Reply:
It looks like a merge issue on main because we stopped hashing packages outside of `tests/data/packages` this file should not contain hash for default_author/contract/stub_0, instead it should be inside `tests/data/packages/hashes.csv`